io/www/src/fragment/replace.js

```diff
@@ -26,6 +26,31 @@ function extractValue(ref) {
     return value.replace(/[\u0000-\u001F\u007F-\u009F]/g, '').replace(/"/g, '\\"');
 }
 
+// Helper function to process entries for a fragment and its parents
+function processDictionaryHierarchy(fragmentId, rootFragment, references, dictionary) {
```

Comment:
could we rename it to something a bit more talkative? collectDictionariesEntries?